docs/2022/puzzles/day15.md

```diff
@@ -6,6 +6,127 @@ import Solver from "../../../../../website/src/components/Solver.js"
 
 https://adventofcode.com/2022/day/15
 
+## Explanation
+
+### Part 1
+
+We first model and parse the input: 
+
+```scala
+case class Position(x: Int, y: Int)
+
+def parse(input: String): List[(Position, Position)] =
+  input.split("\n").toList.map{
+    case s"Sensor at x=$sx, y=$sy: closest beacon is at x=$bx, y=$by" =>
+      (Position(sx.toInt, sy.toInt), Position(bx.toInt, by.toInt))
+  }
+```
+
```

Comment:
Please could you add here a brief overview of the algorithm you are about to implement before you introduce the following snippets